Tools/msi/doc/doc.wixproj

```diff
@@ -10,7 +10,7 @@
     </PropertyGroup>
     <Import Project="..\msi.props" />
     <PropertyGroup>
-        <DocFilename>python$(MajorVersionNumber)$(MinorVersionNumber)$(MicroVersionNumber)$(ReleaseLevelName).chm</DocFilename>
+        <DocFilename>python$(MajorVersionNumber)_$(MinorVersionNumber)$(MicroVersionNumber)$(ReleaseLevelName).chm</DocFilename>
```

Comment:
We should probably separate the minor and micro version of the doc filename too - `python3_10_0.chm` rather than `python3_100.chm`. The actual change ought to be somewhere under Doc, IIRC.